src/library/scala/collection/IterableOnce.scala

```diff
@@ -429,6 +429,17 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
     */
   def span(p: A => Boolean): (C, C)
 
+  /** Applies a (typically) side-effecting function to each element in this collection.
```

Comment:
Is there a non-typical use? I would omit "typically".

Reply:
👍